packages/playwright-test/src/runner.ts

```diff
@@ -346,13 +338,29 @@ export class Runner {
           if (!filterTests(builtSuite, titleMatcher))
             continue;
           projectSuite._addSuite(builtSuite);
-          poolBuilder.buildPools(builtSuite, repeatEachIndex);
         }
       }
     }
     return { rootSuite, fatalErrors };
   }
 
+  private async _loadTests(testFiles: Set<string>): Promise<{ rootSuite: Suite, loadErrors: TestError[] }> {
+    const config = this._configLoader.fullConfig();
+    const testLoader = new TestLoader(config);
+    const loadErrors: TestError[] = [];
+    const rootSuite = new Suite('', 'root');
+    for (const file of testFiles) {
+      const fileSuite = await testLoader.loadTestFile(file, 'loader');
+      if (fileSuite._loadError)
+        loadErrors.push(fileSuite._loadError);
+      // We have to clone only if there maybe subsequent calls of this method.
```

Comment:
Not sure what this comment means today :)

Reply:
Not mine :)